htdocs/fourn/commande/card.php

```diff
@@ -1089,7 +1089,7 @@
 										$lines[$i]->localtax1_tx,
 										$lines[$i]->localtax2_tx,
 										$lines[$i]->fk_product > 0 ? $lines[$i]->fk_product : 0,
-										$productsupplier->product_fourn_price_id,
```

Comment:
If we do'nt use the best supplier price here, it means we must also not use the $productsupplier->ref_supplier just after ?
And also, it means the 
$result = $productsupplier->find_min_price_product_fournisseur(...
that is before is not required ?

This may be used when we create the supplier order from customer order (to buy what we have sell). In such a case, setting price by default to best price is good.
I think we should add a condition to use "0" only when creating from a supplier proposal